tests/unit_tests/test_wave_06.py

```diff
@@ -227,8 +239,9 @@ def test_swap_best_by_category_no_match_is_false():
         my_priority="Clothing",
         their_priority="Clothing"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    assert result == False
```

Comment:
To make this test more robust, you could also check that the length of Tai and Jesse's inventory have not changed. You could also check that the items that were in the lists (before the method under test was executed) are still in the list.